### PR TITLE
Update JavaScript code example

### PR DESCRIPTION
**Pull request by xgirma**
Update JavaScript code example for Selenium WebDriver(JS) of the Browser Automation section. I build and tested the change locally. Here is a repo as a POC for the sample code change. https://github.com/xgirma/cucumber-webdriver-js-browser.

Motivation: The current sample code is not a working example.

### CODE REVIEW

File: content/docs/guides/browser-automation.md [hunk 1]
```diff
@@ -120,28 +120,36 @@ class ExampleSteps: En {
...
+Given('I am on the Google search page', async function () {
+    await driver.get('http://www.google.com');
 });
 
-driver.wait(function() {
-  return driver.getTitle().then(function(title) {
-    return title.toLowerCase().lastIndexOf('cheese!', 0) === 0;
-  });
-}, 3000);
+When('I search for {string}', async function (searchKey) {
```
Comment (luke-hill) [inline content/docs/guides/browser-automation.md]: lets call this searchTerm, not to confuse with the key.return you're using below

File: content/docs/guides/browser-automation.md [hunk 2]
```diff
@@ -120,28 +120,36 @@ class ExampleSteps: En {
...
+    const element = await driver.findElement(By.name('q'));
+    element.sendKeys(searchKey, Key.RETURN);
+    element.submit();
+});
 
-driver.getTitle().then(function(title) {
-  console.log('Page title is: ' + title);
+Then('the page title should start with {string}', {timeout: 60 * 1000}, async function (searchKey) {
```
Comment (luke-hill) [inline content/docs/guides/browser-automation.md]: And also here

**Review by luke-hill — CHANGES_REQUESTED** [review]
Couple of minor changes to avoid name confusion for newer people

File: content/docs/guides/browser-automation.md (hunk 1 above)
Comment (xgirma) [reply content/docs/guides/browser-automation.md]: Thanks, Done.

File: content/docs/guides/browser-automation.md (hunk 2 above)
Comment (xgirma) [reply content/docs/guides/browser-automation.md]: Updated.

**Review by luke-hill — APPROVED** [review]

**Comment by aslakhellesoy** [comment]
Hi @xgirma,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Pull request merged**